nile-frontend/src/components/QA/Questions.jsx

```diff
@@ -1,66 +1,18 @@
 import './QA.css';
-import Grid from '@material-ui/core/Grid';
-import axios from 'axios';
 import './QA.jsx';
-import Answers from './Answers';
+import Question from "./SingleQuestion";
 
 export default function Questions(props) {
-    const displayQuestions = (props) => {
+        console.log(props);
```

Comment:
May want to remove the console logs if you don't need them.

Reply:
just sent a push fixing that